python/paddle/jit/dy2static/error.py

```diff
@@ -120,7 +120,9 @@ def __init__(self, location, function_name):
                 hint_msg = '~' * len(self.source_code[-1]) + ' <--- HERE'
                 self.source_code.append(hint_msg)
                 blank_count.append(blank_count[-1])
-        linecache.clearcache()
+        # Note: It is not possible to modify the contents of the file in the meantime, so there is no need to clear the cache
```

Comment:
```suggestion
        # NOTE(gouzil): It is not possible to modify the contents of the file in the meantime, so there is no need to clear the cache
```

这个 note，说明下 jupyter 下的问题吧，在 note 里说明不能用 `linecache.clearcache` 即可，下面的注释行删掉